src/pages/jobs.md

```diff
@@ -216,12 +264,13 @@ and serve an influx of customers in capacities other than writing software.
 
 Listing a salary range for compliance, we will calibrate compensation to the
 individuals applying. We target 75% of market with our base pay. I see this as a
-career/senior (3-5 years experience) role, involving involving nearly unbounded
-growth potential. This role will be salary/equity compensated, not commission.
+at minimum an early-career role (3+ years experience), involving involving
+nearly unbounded growth potential. More experience applicants are welcome, there
+is plenty to do.  This role will be salary/equity compensated, not commission.
 Our incentives should be aligned to build/grow the business over a long time
 horizon.
 
-130-175k USD salary, 1.5% equity, and
+170-220k USD salary, 1.5% equity, and
```

Comment:
I also need to check these numbers.  I haven't looked at the market. I had planned to do that last week but then got sick.